met-web/src/components/layout/Drawer/CustomDrawer.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import AppBar from '@mui/material/AppBar';
+import { Grid, ListItemButton, List, ListItem, ListItemText, Box } from '@mui/material';
+import { useAppSelector } from '../../../hooks';
+import UserService from '../../../services/UserServices';
+import { useNavigate } from 'react-router-dom';
+
+const Routes = [
+    { name: 'Engagements', path: '/' },
+    { name: 'Surveys', path: '/survey' },
+    { name: 'Calendar', path: '/calendar' },
+    { name: 'Reporting', path: '/reporting' },
+];
+
+const list = (navigate) => (
+    <Box role="presentation">
+        <List>
+            {Routes.map((route, index) => (
+                <ListItem key={route.name}>
+                    <ListItemButton onClick={() => navigate(route.path)}>
+                        <ListItemText
+                            primaryTypographyProps={{ variant: 'h6', sx: { fontWeight: 'bold' } }}
```

Comment:
can this styles be a class ? or is this best way? like fontWeigh bold can it be avoided?